card/mod/email/set/type/email_template.rb

```diff
@@ -78,14 +74,36 @@ def email_config args={}
   config.select { |_k, v| v.present? }
 end
 
+def process_html_message config, args, inline_attachment_url
+  process_message_field(:html_message, config,
+                        args.merge(inline_attachment_url: inline_attachment_url),
+                        "email_html")
+  if config[:html_message].present?
+    config[:html_message] = Card::Mailer.layout config[:html_message]
+  end
+end
+
+
 format do
   view :mail, perms: :none, cache: :never do |args|
-    args = card.email_config(args)
-    text_message = args.delete(:text_message)
-    html_message = args.delete(:html_message)
-    attachment_list = args.delete(:attach)
-    alternative = (text_message.present? && html_message.present?)
-    mail = Card::Mailer.new_mail(args) do
+    config = card.email_config(args)
+    text_message = config.delete(:text_message)
+    attachment_list = config.delete(:attach)
+    email_card = card # card is not accessible in the new_mail block
+    mail = Card::Mailer.new_mail(config) do
```

Comment:
I don't know what kind of black magic I discovered here. I can't access card in this block (because it is called with instance_exec) but email_card is available.